### PR TITLE
[cpp] - buster - EOL

### PR DESCRIPTION
**Pull request by gauravsaini04**
**Devcontainer name**

* CPP

**Changes**

* Changes to manifest.json for CPP dev container image's files for buster EOL

### CODE REVIEW

File: src/cpp/manifest.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
 	"variants": [
 		"bookworm",
```
Comment (samruddhikhandale) [inline src/cpp/manifest.json:4]: Can we cleanup from README file as well?

Comment (samruddhikhandale) [inline src/cpp/manifest.json:4]: Can we cleanup https://github.com/devcontainers/images/blob/fea177cf129ebfe01c7b50263fc1a475f0bd3ea3/src/cpp/.devcontainer/base-scripts/install-vcpkg.sh#L14relevant code ?

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: src/cpp/manifest.json (hunk 1 above)
Comment (gauravsaini04) [reply src/cpp/manifest.json:4]: done

Comment (gauravsaini04) [reply src/cpp/manifest.json:4]: done

**Review by samruddhikhandale — APPROVED** [review]

**Pull request merged**